pkg/p2p/p2p.go

```diff
@@ -21,6 +21,7 @@ type Service interface {
 	Connect(ctx context.Context, addr ma.Multiaddr) (address *bzz.Address, err error)
 	Disconnecter
 	Peers() []Peer
+	BlocklistedPeers() ([]Peer, error)
```

Comment:
It looks a bit strange that function for listing blocklisted peers returns an error, but list of peers does not.

Reply:
true. but `Peers` does not involve any storage operations, therefore does not return a possible error :+1: